resources/user-agents/browsers/nokia/nokia.json

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
I removed this pattern because I feel that it masks other detection issues.

It was good for detecting agents that started with "Nokia" that the `nokia-browser.json` file wasn't catching, but it would also detect patterns that should have been something else, like UC Browser or IE Mobile just because they had the string "nokia" in them.

This may cause some agents to detect as "Default Browser" that would have detetced as "Nokia", but I think that's better as it will be more obvious that a pattern is missing.